### PR TITLE
S01US29

### PR DESCRIPTION
**Pull request by Mewlyscin**
create navbar
page : UserCompte and MyRecipe
update main: router /UserCompte and MyRecipe

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,40 +1,18 @@
-import Counter from "./components/Counter";
-import logo from "./assets/logo.svg";
+// router react
```
Comment (PierreA44) [inline frontend/src/App.jsx:1]: Useless

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -1,40 +1,18 @@
...
+
+import { Outlet } from "react-router-dom";
+
+// my imports
```
Comment (PierreA44) [inline frontend/src/App.jsx:5]: Useless

File: frontend/src/components/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { NavLink } from "react-router-dom";
+
+// import logo
```
Comment (PierreA44) [inline frontend/src/components/Navbar.jsx:3]: Useless

File: frontend/src/components/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+import logo from "../assets/logo_et_icones/logo1.png";
+
+// my function nav
```
Comment (PierreA44) [inline frontend/src/components/Navbar.jsx:7]: Useless

File: frontend/src/components/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+function Navbar() {
+  return (
+    <div className="bg-orangep flex flex-row justify-between">
+      <NavLink className="m-2" to="/home">
```
Comment (PierreA44) [inline frontend/src/components/Navbar.jsx:12]: this path does'nt exist

File: frontend/src/main.jsx [hunk 6]
```diff
@@ -1,15 +1,30 @@
-import React from "react";
-import ReactDOM from "react-dom/client";
+// import react
```
Comment (PierreA44) [inline frontend/src/main.jsx:1]: Useless

File: frontend/src/main.jsx [hunk 7]
```diff
@@ -1,15 +1,30 @@
...
 
+import React from "react";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import ReactDOM from "react-dom/client";
 
 import App from "./App";
+import MyRecipe from "./pages/MyRecipe";
```
Comment (PierreA44) [inline frontend/src/main.jsx:8]: MyRecipe isn't a good syntax... use RecipePage

File: frontend/src/main.jsx [hunk 8]
```diff
@@ -1,15 +1,30 @@
...
+import UserCompte from "./pages/UserCompte";
```
Comment (PierreA44) [inline frontend/src/main.jsx:9]: French and English in same namefile... bad... use UserAccountPage or UserPage

File: frontend/src/main.jsx [hunk 9]
```diff
@@ -1,15 +1,30 @@
...
+
+// imports page
```
Comment (PierreA44) [inline frontend/src/main.jsx:11]: Useless

File: frontend/src/main.jsx [hunk 10]
```diff
@@ -1,15 +1,30 @@
...
+
+// router
```
Comment (PierreA44) [inline frontend/src/main.jsx:13]: Useless

**Review by PierreA44 — CHANGES_REQUESTED** [review]
Fix many problems

**Review by QuentinVerwaerde — CHANGES_REQUESTED** [review]
Nice but like said before me, some changes are necessary : 
names should be in english only
no commented text in generally allowed 
you can't use a path that does not exist in a Navlink : you'll need to create it or modify it

**Review by PierreA44 — APPROVED** [review]
nice work. thanks for changes

**Review by bayram-69 — APPROVED** [review]

**Review by akornne — APPROVED** [review]

File: .gitignore [hunk 11]
```diff
@@ -2,4 +2,5 @@
 node_modules
 .DS_Store
 .idea
-.vscode
\ No newline at end of file
+.vscode
+.env
```
Comment (ioayoub) [inline .gitignore:6]: updating this file is not allowed, you already have .gitgnore file on both front and backend folders

File: frontend/src/pages/RecipePage.jsx [hunk 12]
```diff
@@ -0,0 +1,9 @@
+function MyRecipe() {
```
Comment (ioayoub) [inline frontend/src/pages/RecipePage.jsx:1]: Empty pages are not allowed.

File: frontend/src/pages/UserAccountPage.jsx [hunk 13]
```diff
@@ -0,0 +1,9 @@
+function UserAccountPage() {
```
Comment (ioayoub) [inline frontend/src/pages/UserAccountPage.jsx:1]: Empty pages are not allowed.

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Review by ioayoub — APPROVED** [review]

**Pull request merged**